src/components/Divider.scss

```diff
@@ -0,0 +1,38 @@
+//------------------------------------------------------------//
+
+// * Dividers
+// * 1. chapter
```

Comment:
Oh, nothing, I just have a snippet saved as a template for my CSS comments and was rushing at the end of this.
```

//------------------------------------------------------------//

// * Title
// * 1. chapter

// Notes:
// * none

// TODO : none

//------------------------------------------------------------//

// * 1. Chapter

```